src/rgw/rgw_user.cc

```diff
@@ -1735,6 +1735,7 @@ int RGWUser::init(RGWUserAdminOpState& op_state)
   }
 
   if (!uid.empty() && (uid.compare(RGW_USER_ANON_ID) != 0)) {
+    op_state.set_reserved_user(false);
```

Comment:
Here we are checking for the reserved user and avoids the anonymous user. So the value of `reserved_user` should be true logically ?